.github/workflows/cd.yml

```diff
@@ -61,3 +61,38 @@ jobs:
         shell: bash
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_PUBLISH_TOKEN }}
+
+      - name: Configure AWS credentials for public artifacts account
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::537451838446:role/public-cicd-remote-deployer
+          aws-region: us-west-2
+          aws-access-key-id: ${{ secrets.PUBLIC_CICD_DEPLOYER_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.PUBLIC_CICD_DEPLOYER_SECRET_ACCESS_KEY }}
+          role-duration-seconds: 1200
+          role-session-name: javascript-sdk-lambda-layer-deployer
+
+      - name: Publish lambda layer
+        run: |
+          export VERSION="${{ needs.release.outputs.version }}"
+          echo "Publishing lambda layer with version $VERSION"
+          mkdir -p nodejs/node_modules
+          pushd nodejs/node_modules
+            npm i @gomomento/sdk@${VERSION}
+          popd
+          zip -r layer.zip nodejs
+          LAYER_NAME="momento-sdk-javascript"
+          LAYER_VERSION=$(aws lambda publish-layer-version \
```

Comment:
Is there some way that we can explictly specify the layer version so that we can maintain some sort of mapping between the layer version number and our SDK version number?

Reply:
Unfortunately we can't do that :sad-vir:. Amazon auto increments the version of the lambda layer for us. It also has to be a whole number. I can see if I can tag the layer with the version of the sdk its using